Makefile

```diff
@@ -31,16 +31,9 @@ help:  ## Display this help
 ################################ make install #################################
 .PHONY: install
 install: ## Installs all dependencies needed to compile Scorecard
-install: | $(GINKGO) $(GOLANGGCI_LINT) $(PROTOC_GEN_GO) $(PROTOC)
-
-$(GINKGO):
-	go get -u github.com/onsi/ginkgo/ginkgo@v1.16.2
-
-$(GOLANGGCI_LINT):
-	go install github.com/golangci/golangci-lint/cmd/golangci-lint@v1.40.0
-
-$(PROTOC_GEN_GO):
-	go install google.golang.org/protobuf/cmd/protoc-gen-go
+install: | $(PROTOC) 
+	@echo Installing tools from tools.go
```

Comment:
How about this?

```
install:
    go get ./...
```

Reply:
>go install, with or without a version suffix (as described above), is now the recommended way to build and install packages in module mode. go get should be used with the -d flag to adjust the current module's dependencies without building packages, and use of go get to build and install packages is deprecated. In a future release, the -d flag will always be enabled.

https://tip.golang.org/doc/go1.16#go-command

`go install` is the recommended way. That is the reason. Let me know.

Reply:
Thanks